.github/workflows/test-upstream.yml

```diff
@@ -70,6 +76,11 @@ jobs:
           channels: dask/label/dev,conda-forge,nodefaults
           activate-environment: dask-sql
           environment-file: continuous_integration/environment-3.9-dev.yaml
+      - name: Optionally update upstream cargo dependencies
+        if: github.event.schedule == '0 3 * * *'
```

Comment:
Just noting that depending on if we want to test _only_ nightly DF or both nightly DF and nightly Dask, we might want to add a similar condition to the `Install upstream dev dask-ml` step for the other `cron` schedule